src/lib/lightning/lnd/lndService.ts

```diff
@@ -186,6 +186,21 @@ class LndService implements LightningService {
     );
   }
 
+  async addListenerToNode(node: LightningNode): Promise<void> {
```

Comment:
As I mentioned in a comment above, there really isn't a need to add/remove the listener for LND nodes. But you will need to keep these functions declared here in order to conform to the `LightningService` interface. They can just be empty functions that do not do anything in this case.